paypal/paypal-messaging-components
# feat: frites modals en variants

## Description
support passage of language query param to determine display of content in english or in native language for FRITES modals

## Screenshots
<img width="638" alt="Screenshot 2023-09-08 at 9 45 54 AM" src="https://github.com/paypal/paypal-messaging-components/assets/71471412/7de99fbe-0904-4859-9044-aff0f0fd624d">

## Testing instructions
Make request without language param, modal should display in default, i.e. native, language.
Make request with 'language' query param in the BCP-47 format (e.g. en-IT) appropriate for english version of FRITES modal. modal should display in english.
Make request with 'language' query param that is not appropriate for FRITES. Response should be in default language.
